cli/src/views/App/Price/components/Price.jsx

```diff
@@ -208,14 +214,20 @@ export default function Price({ notify, setLoading }) {
               key: 'action',
               render: (text, record) => (
                 <span>
-                  <PopConfirm
-                    title='Are you sure to delete this genre?'
-                    task={hdRemove.bind(this, record._id)}
-                    okText='Sure, remove it'
-                    cancelText='Not now'
-                  >
-                    <span className='gx-link'>Delete</span>
-                  </PopConfirm>
+                  <span className='gx-link' onClick={onShowModal}>Delete</span>
+                  {
+                    modal && <SweetAlert
+                      warning
+                      showCancel
+                      confirmBtnText='Yes, delete it !'
+                      cancelBtnBsStyle='default'
+                      title='Are you sure to delete ?'
+                      onConfirm={hdRemove.bind(this, record._id)}
+                      onCancel={onShowModal}
+                    >
+                      <span>You will not be able to recover this Price</span>
+                    </SweetAlert>
+                  }
```

Comment:
The alert is always in 'warning' type, so we can hard-code true for this 'warning' prop. Moreover, some of the props such as 'confirmBtnText', 'cancelBtnBsStyle', 'title', 'showCancel' are the same in all the cases so i suggest you need to have a better way to reuse those value.

Reply:
Updated component SweetAlert to re-use in People and Price